src/leetCodeManager.ts

```diff
@@ -117,6 +117,19 @@ class LeetCodeManager extends EventEmitter {
     public getUser(): string | undefined {
         return this.currentUser;
     }
+
+    private tryParseUserName(output: string): string {
+        const lines: string[] = output.trim().split(/\r?\n/);
+        if (lines.length === 3) {
+            const reg: RegExp = /^\s*.{1}\s*(.+)\s*https:\/\/leetcode/;
```

Comment:
The regex executing on 
```
'    ×      vigilans             https://leetcode-cn.com'
```
will return
```
'vigilans             '
```
Append `?` to `.+` to prevent greedy match could solve the problem.

Besides, is there any difference between `.{1}` and `.`?

Reply:
Updated